cmd/login.go

```diff
@@ -234,9 +234,10 @@ const ReadHeaderTimeoutSeconds = 3
 
 // loginCmd represents the login command.
 var loginCmd = &cobra.Command{ //nolint:gochecknoglobals
-	Use:   "login",
-	Short: "Log in to an Ampersand account",
-	Long:  "Log in to an Ampersand account.",
```

Comment:
Not a big deal, but `amp help login` still works, just wanted to highlight the scope of 'Hidden'

Reply:
Yeah that's intended. This is about reducing confusion, not actually disabling functionality. If we can get clerk auth working in prod, I think that's still the preferred approach.